src/libpgagroal/configuration.c

```diff
@@ -75,9 +75,10 @@ static void copy_user(struct user* dst, struct user* src);
 static int restart_int(char* name, int e, int n);
 static int restart_string(char* name, char* e, char* n);
 static int restart_limit(char* name, struct configuration* config, struct configuration* reload);
+static int restart_server(struct server* src, struct server* dst);
 
 static bool is_empty_string(char* s);
-
+static bool is_same_connection(struct server* s1, struct server* s2);
```

Comment:
`is_same_server` is better

Reply:
Function name changed. Also added some documentation.